system_tests/vision.py

```diff
@@ -493,19 +493,14 @@ def _assert_properties(self, image_property):
             self.assertNotEqual(color_info.score, 0.0)
 
     def test_detect_properties_content(self):
-        self._pb_not_implemented_skip(
-            'gRPC not implemented for image properties detection.')
         client = Config.CLIENT
         with open(FACE_FILE, 'rb') as image_file:
             image = client.image(content=image_file.read())
         properties = image.detect_properties()
-        self.assertEqual(len(properties), 1)
-        image_property = properties[0]
-        self._assert_properties(image_property)
+        self.assertEqual(len(properties[0]))
```

Comment:
Did you mean to change the actual output type? This is a "breaking API change" that is not at all related to "Add gRPC support for image properties."